pypesto/sampling/sample.py

```diff
@@ -62,12 +63,17 @@ def sample(
     # initialize sampler to problem
     sampler.initialize(problem=problem, x0=x0)
 
-    # perform the sampling
+    # perform the sampling and track time
+    t_start = process_time()
     sampler.sample(n_samples=n_samples)
+    t_elapsed = process_time() - t_start
 
     # extract results
     sampler_result = sampler.get_samples()
 
+    # record time
+    sampler_result['elapsed_time'] = t_elapsed
+
```

Comment:
should one maybe compute the burn-in here as well automatically? and set it in the result. otherwise the field will not be set, it I see correctly ...

Reply:
When the sampling is run, it is set to zero automatically.
Then the calculation of convergence test is optional. This is why I had separate them.